projects/plugins/jetpack/tests/action-test-coverage.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-set -eo pipefail
+set -o pipefail
```

Comment:
This will prevent Jetpack test failures from making the job fail. That's probably not what we want.

Reply:
Why? In my tests, script exit code would be maintained, so if one of the script commands will fail, the resulting exit code would be non-0.

Reply:
Taking out the `-e` makes it not exit when something fails, so it winds up exiting with the status of the last command run. That's probably going to be the `echo "::endgroup::"` which is always going to succeed.